src/chrome/collections/utilities.ts

```diff
@@ -15,3 +19,11 @@ export function determineOrderingOfStrings(left: string, right: string): number
         return 0;
     }
 }
+
+export function singleOne<T>(array: ReadonlyArray<T>): T {
```

Comment:
Nit: Consider renaming this function to something more descriptive (something of sorts singleElementArrayValue() ? )

Reply:
Renamed to: singleElementOfArray